Mos/Localization/Localizable.swift

```diff
@@ -5,29 +5,28 @@
 //  Created by Caldis on 2018/2/25.
 //  Copyright © 2018年 Caldis. All rights reserved.
 //
-
 import Cocoa
 
 class i18n {
-    
+
```

Comment:
Do not change author's code style, just follow it. Propose a better style in another commit if necessary.

Reply:
My XCode is set to remove extra whitespaces on save. As you can see they are not needed.

Reply:
These insignificant changes should be referred to separately as a PR. Bug, this is not a big problem. These spaces are really useless.